org.civicrm.reqangular/src/common/services/api.js

```diff
@@ -84,6 +85,9 @@ define([
           }.bind(this))(),
           (function () {
             var params = _.assign({}, filters, { 'return': 'id' });
+            params = _.omit(params, function (value, key) {
+              return key.startsWith('api.');
```

Comment:
`String.prototype.startsWith()` is not available on IE, please always check the compatibility (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/startsWith)

Use [`_.startsWith`](https://lodash.com/docs/4.17.4#startsWith)